clients/vscode-hlasmplugin/src/test/syntaxes/endevorLong/parenthesis.listing.snap

```diff
@@ -142,8 +142,8 @@
 #                                                                    ^ source.hlasmListing
 #                                                                     ^^^ source.hlasmListing punctuation.definition.tag.hlasm
 #                                                                        ^^^^^^^^^^^^^^ source.hlasmListing
-#                                                                                      ^^^^^^^^^^^ source.hlasmListing comment.hlasm
-#                                                                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasmListing
+#                                                                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasmListing comment.hlasm
```

Comment:
Same thing. Spaces before `REMARK(REM)`, except the one right after `OP1,OP2` should be part of the comment.